tests/unit/test-cmd/test.build.js

```diff
@@ -329,4 +369,41 @@ describe('build', () => {
 
   });
 
+  describe('defaultPackageCreator', () => {
+
+    it('should reject on Unexpected errors', () => withTempDir(
+      (tmpDir) => {
+        const fakeEventToPromise = sinon.spy(async (stream) => {
+          await defaultEventToPromise(stream, 'close');
+          const files = await fs.readdir(tmpDir.path());
```

Comment:
we should add an inline comment here to explain why we are removing the content of the tmpDir here.